docs/rules/enforce-memoize-getters.md

```
@@ -0,0 +1,45 @@
+
+## Notes
+- Applies only to private instance getters (`get` accessors with `private` accessibility`).
+- Ignores static getters.
+- Recognizes `@Memoize`, `@Memoize()`, and namespaced forms like `@ns.Memoize()`.
+- Auto-fix adds `@Memoize()` and imports `Memoize` from `@blumintinc/typescript-memoize` if missing, without duplicating existing imports or aliases.
+- For ephemeral getters by design, use a targeted disable comment on the getter.
+
+```ts
+// eslint-disable-next-line @blumintinc/blumint/enforce-memoize-getters -- ephemeral by design
+private get timestamp() { return Date.now(); }
+```
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)